rules/aws/credential_access_aws_iam_assume_role_brute_force.toml

```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/07/16"
+ecs_version = ["1.5.0"]
+maturity = "development"
+updated_date = "2020/07/16"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a high number of failed attempts to assume an AWS IAM role. IAM roles are used to delegate access to users or
+services. An adversary may attempt to enumerate IAM roles in order to determine whether a role exists or not before
+attempting to assume or hijack the discovered role.
+"""
+index = ["filebeat-*"]
+language = "kuery"
+license = "Elastic License"
+name = "AWS IAM Assume Role Brute Force"
+references = [
+    "https://www.praetorian.com/blog/aws-iam-assume-role-vulnerabilities",
+    "https://rhinosecuritylabs.com/aws/assume-worst-aws-assume-role-enumeration/",
+]
+risk_score = 47
+rule_id = "ea248a02-bc47-4043-8e94-2885b19b2636"
+severity = "medium"
+tags = ["AWS", "Elastic"]
```

Comment:
```suggestion
tags = ["AWS", "Elastic", "IAM"]
```

Reply:
Are we being more open with the tags? We didn’t add the services as a tag for the other AWS rules. The tags populate in the UI and we probably don’t want it to get messy with one off tags having just a single rule. I think we should keep them consistent at the least.

Reply:
💯 for consistency

Reply:
good catch. makes more sense to do all of those in one fell swoop if we do

Reply:
Are you suggesting that we tag all Identity & Access Management (IAM)-type rules with `IAM`, not just the AWS ones? If so, will be start tagging other rules with the service-type as well as the platform (AWS, Okta, Azure, etc).

Reply:
Definitely worth hashing out the outcome of our conversation from earlier today on tags and categorization

Reply:
Thank you for your comments, all. I'll resolve this conversation with the plan that we'll revisit tags on our rules after we've spoken to the product team.